user_sync/config.py

```diff
@@ -312,6 +307,40 @@ def get_rule_options(self):
         if (new_account_type == None):
             new_account_type = user_sync.identity_type.ENTERPRISE_IDENTITY_TYPE
             self.logger.warning("Assuming the identity type for users is: %s", new_account_type)
+
+        after_mapping_hook = None
+        extended_attributes = None
+        extensions_config = self.main_config.get_list_config('extensions', True)
+        if (extensions_config != None):
```

Comment:
I used the equality when in the midst of a block of code that was already using it; otherwise I did what you're calling the right way. Should be addressed globally.